frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -864,3 +886,33 @@ func runAcloudSetup(execContext ExecContext, artifactsRootDir, artifactsDir, run
 	go run(execContext(context.TODO(), "sudo", "-u", cvdUser, "chmod", "644", runtimeDir+"/cuttlefish_runtime/logs/metrics.log"))
 	go run(execContext(context.TODO(), "sudo", "-u", cvdUser, "chmod", "644", runtimeDir+"/cuttlefish_runtime/logs/modem_simulator.log"))
 }
+
+func sliceItoa(s []uint32) []string {
+	result := make([]string, len(s))
+	for i := range s {
+		result[i] = strconv.Itoa(i)
+	}
```

Comment:
This looks wrong, it should be:
```
for i, v := range s {
    result[i] = strconv.Itoa(v)
}
```

Reply:
worth a unit test :)

Reply:
good catch! done, added some unit tests.